suite-native/graph/src/hooks.ts

```diff
@@ -104,15 +110,19 @@ export const useGraphForSingleAccount = ({
         selectIsElectrumBackendSelected(state, account?.symbol ?? 'btc'),
     );
 
+    const graphForAccounts = useGraphForAccounts({
+        accounts,
+        fiatCurrency,
+        startOfTimeFrameDate,
+        endOfTimeFrameDate,
+        isPortfolioGraph: false,
+        isElectrumBackend,
+    });
+
+    useEffect(() => checkAndReportGraphError(graphForAccounts.error), [graphForAccounts.error]);
```

Comment:
This whole useEffect could be hook, but it's just a nit

Reply:
I don't think it's worth to create hook for one liner.

Reply:
I am in favour of refactoring more of shared code for graph fetching IF we ever touch this graph functionality again.   this feels almost the same, since it is just one line of code. I'll keep it this way. Thanks!

Reply:
Yeah, sure. It's just a nitpick and as you say, it's dependant on IF we touch it again, so leave it as is